sbi/inference/base.py

```diff
@@ -7,7 +7,7 @@
 from torch.utils.tensorboard import SummaryWriter
 
 from copy import deepcopy
-from sbi.simulators.simutils import simulate_in_batches
+from sbi.simulators.simutils import simulate_in_batches, simulate_mp_in_batches
```

Comment:
I am still unhappy with this name. but if you wanna go with it, it's fine.

Reply:
I think you are commenting on an old version somehow. This should now be called `simulate_in_batches_mp`. Your other comment should also be fixed ;)

Reply:
OK thanks. this is so annoying that github shows an outdated version although I refreshed the page. Do you know a way to change this?